src/styles/styles.js

```diff
@@ -1027,6 +1027,10 @@ const styles = {
         },
     },
 
+    floatingActionButtonContainer: {
+        bottom: 0,
+    },
+
```

Comment:
I see this style being added but not used anywhere in this PR nor in our codebase, should this be here?

Reply:
removed it, good catch, thx